### PR TITLE
fix: Simple Learn logo being stretched

### LINKED ISSUE #133: [BUG] Simple Learn logo is stretched
### Description of the bug

The Simple Learn logo is stretched which makes it look bad.

### Please add screenshots (if applicable)

image

### Add any other context about the problem here

Already have the fix for this. 🙂

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Panquesito7**
## Related Issue 

- Information about the related issue 

Closes #133 

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

Tested in the web; works.

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

 Original           | Updated
 :--------------------: |:--------------------:
image | image |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/web3community/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/web3community/web3community.github.io/pull/134"></a>

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -111,7 +111,7 @@ <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-80
       <div
         class="cursor-pointer  bg-white flex-1 rounded-md shadow focus:outline-none focus:shadow-outline transform transition hover:scale-105 hover:shadow-lg duration-300 ease-in-out p-4">
         <div class="flex items-center justify-start overflow-hidden">
-          <img alt="simplilearn" class="bg-gray-50 p-5 rounded-md" src="./static/img/simplilearn.png" />
+          <img alt="simplilearn" class="bg-gray-50 p-5 rounded-md" src="https://github.com/web3community/web3community.github.io/tree/fix_simplelilearn_logo" />
```
Comment (krishguptadev) [inline src/index.html:114]: Wait? How does this work?

Comment (Panquesito7) [reply src/index.html:114]: > Wait? How does this work?

I fixed this in another commit.

Comment (krishguptadev) [reply src/index.html:114]: so why did you make a PR that does nothing and merge it? 😅

Comment (Panquesito7) [reply src/index.html:114]: > so why did you make a PR that does nothing and merge it? 😅

Because I didn't notice that bug.

**Pull request merged**